micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/DiskSpaceMetrics.java

```diff
@@ -34,28 +36,40 @@
  *
  * @author jmcshane
  * @author Johnny Lim
+ * @author Chris Bono
  */
 @Incubating(since = "1.1.0")
 @NonNullApi
 @NonNullFields
 public class DiskSpaceMetrics implements MeterBinder {
+
     private final Iterable<Tag> tags;
-    private final File path;
-    private final String absolutePath;
+    private final List<File> paths;
 
     public DiskSpaceMetrics(File path) {
         this(path, emptyList());
     }
 
+    public DiskSpaceMetrics(List<File> paths) {
+        this(paths, emptyList());
+    }
+
     public DiskSpaceMetrics(File path, Iterable<Tag> tags) {
```

Comment:
I was hoping to not add 2 new constructors by editing the existing ones and changing their single `path` param to a varargs path param. That approach came up short w/ this particular `<init>` as varargs has to be the last in param list.  

I am open to suggestions. I think a builder would be overkill. What are thoughts on deprecation of the single path `<init>` variants?